examples/secureTokenAndroidKeyStore/build.gradle

```diff
@@ -30,7 +30,7 @@ dependencies {
     })
     compile 'com.android.support:appcompat-v7:25.2.0'
     testCompile 'junit:junit:4.12'
-    compile 'io.realm:android-secure-userstore:1.0.0'
+    compile 'io.realm:secure-userstore:1.0.1'
```

Comment:
https://bintray.com/realm/maven/secure-userstore/1.0.1 I just visted this link. nvm